OtripleS.Portal.Web/Models/Teachers/Exceptions/NullTeachertException.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+
+namespace OtripleS.Portal.Web.Models.Teachers.Exceptions
+{
+    public class NullTeachertException : Exception
+    {
+        public NullTeachertException()
+            : base("Null teacher error occurred.") { }
```

Comment:
`"Null teacher error occurred."` to `message: "Null teacher error occurred."` 
Also for all the other messages